WORKSPACE

```diff
@@ -101,26 +101,26 @@ go_repository(
 
 go_repository(
     name = "io_k8s_sigs_kind",
-    commit = "1284e993a84a56c994ea541dbcb97486a7b86b50",
     importpath = "sigs.k8s.io/kind",
+    tag = "0.1.0",
 )
 
 go_repository(
     name = "io_k8s_sigs_kustomize",
-    commit = "58492e2d83c59ed63881311f46ad6251f77dabc3",
     importpath = "sigs.k8s.io/kustomize",
+    tag = "v2.0.3",
```

Comment:
Pin this to `v1.0.11`. There are breaking changes in kustomize 2.x that we'll need to investigate before we can migrate over.

Reply:
For reference: https://github.com/kubernetes-sigs/kustomize/blob/master/docs/version2.0.0.md

Reply:
Thanks! Done